ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -2159,6 +2169,18 @@ FCU_DFSS_DFRMM_CHWBZC:
   - DFRMM
   - CHWBZC
 
+FCU_SS_DFSS_ZTC:
```

Comment:
Not acceptable - the chilled water valve should be modeled via an abstract type (e.g., CHWZTC)

Reply:
The chilled water command is a status point with state open /close but the command in the abstract type CHWZTC is a percentage command

Reply:
I have changed this FCU type to non canonical